### PR TITLE
Narrow accepted character range (Fixes #187 and #213)

### LINKED ISSUE #187: Don't allow Japanese in meaning
If in a review, such as 一つ, the user types ひとつ, or any other Japanese characters, for the meaning, don't allow it to be submitted.

### PR DESCRIPTION
**Pull request by turtlemaster19**
To fix #187 and #213, this code attempts to narrow the valid character range.

### CODE REVIEW

File: ios/AnswerChecker.swift [hunk 1]
```diff
@@ -23,24 +23,27 @@ import Foundation
...
-      "さしすせそざじずぜぞ" +
-      "たちつてとだぢづでど" +
-      "なにぬねの" +
-      "はひふへほばびぶべぼぱぴぷぺぽ" +
-      "まみむめも" +
-      "らりるれろ" +
-      "やゆよゃゅょぃっ" +
-      "わをん")
+  @objc static let kAsciiCharacterSet = CharacterSet(charactersIn: Unicode.Scalar(0x00) ..< Unicode.Scalar(0x7F)!)
+  @objc static let kKanaCharacterSet = CharacterSet(charactersIn: Unicode.Scalar(UInt32(0x3040))! ..< Unicode.Scalar(UInt32(0x309D))!)
```
Comment (davidsansome) [inline ios/AnswerChecker.swift]: This should be called kHiraganaCharacterSet?

Comment (turtlemaster19) [reply ios/AnswerChecker.swift]: Agree. I didn't want to change your name. Will change it, though.

**Comment by turtlemaster19** [comment]
@davidsansome Ready to merge.

File: ios/AnswerChecker.swift (hunk 1 above)
Comment (davidsansome) [reply ios/AnswerChecker.swift]: Oops that was my fault, I didn't notice :smile:  Thanks!

**Pull request merged**